### DISCUSSION TITLE
Tab to show directory contents

### DISCUSSION THREAD

**Discussion by koalazub** [starship/starship#2110] [General]
Hi!

I wanted to know if there was a means of being able to show the contents by hitting the tab key like when we're trying to `cd` into something?

**Comment by mickimnet** [comment]
Hello @koalazub,

What contents are you referring to? Do mean the prompt, the options of `Starship` or something else on the command line?

**Reply by koalazub** [reply]
By contents I mean the contents of a directory an example would be when I'm typing `cd ~/.config` and I then hit `tab`, I would hope that I could view the contents of the directory

**Reply by koalazub** [reply]
ahhh nvm. I just realised that this was a Nushell issue I was having and not a Starship related problem 🤦🏾 

In case you wanted clarification on what my problem was in Nushell, it was about the completion characteristic where I'd hit tab and expected a list to pop down with the contents of the directory. 

Didn't realise that `completion_type` attribute was set to `circular` instead of `listed`. 


Please let me know if you'd like me to delete this whole post!

**Reply by mickimnet** [reply]
`Starship` is only responsible for the contents of the prompt and not with the command line/readline.

Don‘t delete the post. Someone else may have the same question and find this information useful.
